cli/tests/testdata/badly_linted.mjs

```diff
@@ -0,0 +1,2 @@
+// Deliberately using .mjs to avoid triggering deno lint on ci
+let a = 5;
```

Comment:
Put these files in `cli/tests/testdata/lint` (or a subdirectory in that file) to avoid the problem from the comment

Reply:
I thought deno lint will linting all js files, so I did the same trick in https://github.com/denoland/deno/blob/2cc1577d286814d941000fc4653f6e8d59b2e190/cli/tests/testdata/badly_formatted.mjs#L1